tests/test_room_message_timestamp.py

```diff
@@ -45,14 +45,14 @@ def test_can_find_last_message_timestamp(self) -> None:
         def send_message_and_assert_latest_activity(room, message, tok) -> None:
             body = self.helper.send(room, message, tok=tok)
 
-            event_id = self.helper.get_event(room, body.get("event_id"), tok=tok)
+            event_id = self.helper.get_event(room, body.get("event_id", ""), tok=tok)
```

Comment:
This one is on me when I was working it. This stuff is misnamed. I blame looking at it for months for everything being blurry :laughing: 
```suggestion
            event_id = body.get("event_id")
            assert event_id is not None
            self.helper.get_event(room, event_id, tok=tok)
```
Something like that makes more sense?